libraries/AP_Mount/AP_Mount_Siyi.h

```diff
@@ -0,0 +1,195 @@
+/*
+  Siyi gimbal driver using custom serial protocol
+
+  Packet format (courtesy of Siyi's SDK document)
+
+  -------------------------------------------------------------------------------------------
+  Field     Index   Bytes       Description
+  -------------------------------------------------------------------------------------------
+  STX       0       2           0x5566: starting mark
+  CTRL      2       1           bit 0: need_ack.  set if the current data packet needs ack
+                                bit 1: ack_pack.  set if the current data packate IS an ack
+                                bit 2-7: reserved
+  Data_len  3       2           Data field byte length.  Low byte in the front
+  SEQ       5       2           Frame sequence (0 ~ 65535).  Low byte in the front.  May be used to detect packet loss
+  CMD_ID    7       1           Command ID
+  DATA      8       Data_len    Data
+  CRC16             2           CRC16 check the complete data package.  Low byte in the front
+ */
+
+#pragma once
+
+#include "AP_Mount_Backend.h"
+
+#ifndef HAL_MOUNT_SIYI_ENABLED
+#define HAL_MOUNT_SIYI_ENABLED HAL_MOUNT_ENABLED
+#endif
+
+#if HAL_MOUNT_SIYI_ENABLED
+
+#include <AP_HAL/AP_HAL.h>
+#include <AP_AHRS/AP_AHRS.h>
```

Comment:
Is this include required?

Reply:
Not required in the .h (just the .cpp) so moved.txs